task/maven/0.1/README.md

```diff
@@ -12,6 +12,8 @@ kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/
 
 - **GOALS**: Maven `goals` to be executed
 - **MAVEN_MIRROR_URL**: Maven mirror url (to be inserted into ~/.m2/settings.xml)   
+- **SERVER_USER**: Username to to authenticate to the server (to be inserted into ~/.m2/settings.xml)
```

Comment:
```suggestion
- **SERVER_USER**: Username to authenticate to the server (to be inserted into ~/.m2/settings.xml)
```

Reply:
@vinamra28 
Embarrassing. I fixed the typos.